exercises/leap/LeapSpec.groovy

```diff
@@ -0,0 +1,25 @@
+@Grab('org.spockframework:spock-core:1.0-groovy-2.4')
+import spock.lang.*
+
+class LeapSpec extends Specification {
+
+    def 'a year not divisible by 4 is not a leap year'() {
+        expect: new Year(2015).isLeapYear() == false
```

Comment:
Hey, @kytrinyx .  Really sorry that it's taken me so long to get back to this.

Looks like, in the case that all years report as being leap years.
```
] groovy LeapSpec.groovy 
JUnit 4 Runner, Tests: 1, Failures: 1, Time: 112
Test Failure: a year not divisible by 4 is not a leap year(LeapSpec)
Condition not satisfied:

new Year(2015).isLeapYear() == false
|              |            |
Year@19553973  true         false

        at LeapSpec.a year not divisible by 4 is not a leap year(LeapSpec.groovy:7)
```
It looks like the message if the `.isLeapYear()` method doesn't exist isn't quite  as nice.
```
] groovy LeapSpec.groovy 
JUnit 4 Runner, Tests: 1, Failures: 1, Time: 99
Test Failure: a year not divisible by 4 is not a leap year(LeapSpec)
groovy.lang.MissingMethodException: No signature of method: Year.isLeapYear() is applicable for argument types: () values: []
Possible solutions: setYear(java.lang.Integer), getYear()
        at LeapSpec.a year not divisible by 4 is not a leap year(LeapSpec.groovy:7)
```

I agree with you: I don't think that either one gives the learner a good cue as to what might be a next step.  I'm not exactly sure how to make this better at the moment, but I'll give it some more thought.  (And any help would be greatly appreciated, of course. 😸 ).

One other thing I noticed is that the example code seems to cover more test cases than we cover (looking at false, divisible-by-four non-leap-years).  That's not so great— I'll have a look at the common tests: if they aren't testing against the additional leap year conditions, I'm thinking that we might want to simplify `Example.groovy`.